pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties

```diff
@@ -102,11 +103,15 @@ org.osgi.framework.system.packages.extra= \
  javax.ws.rs.*;version\="1.1.1", \
  org.dom4j.*, \
  org.dom4j, \
-org.pentaho.metadata.*, \
+ org.pentaho.metadata.*, \
  org.pentaho.pms.core.*, \
  org.pentaho.pms.schema.*, \
  org.pentaho.platform.servicecoordination.api;version\="7.1.0", \
- org.pentaho.platform.servicecoordination.impl;version\="7.1.0"
+ org.pentaho.platform.servicecoordination.impl;version\="7.1.0", \
+ org.owasp.encoder.*, \
+ io.reactivex.*;version\="2.0.4", \
```

Comment:
We really need to start using properties for these versions. The file is filtered. Anyway not your fault.